camel-core/src/main/java/org/apache/camel/builder/RouteBuilder.java

```diff
@@ -53,6 +53,7 @@
     private AtomicBoolean initialized = new AtomicBoolean(false);
     private RestsDefinition restCollection = new RestsDefinition();
     private Map<String, RestConfigurationDefinition> restConfigurations;
+    private List<TransformerBuilder> transformerBuilders = new ArrayList<TransformerBuilder>();
```

Comment:
Optional: no need to repeat the TransformerBuilder in new ArrayList<TransformerBuilder>();